Zemismart/__init__.py

```diff
@@ -174,9 +250,64 @@ def save_battery(self, battery):
         self.battery = battery
 
     def update(self):
+        self._get_position_responses = set()
         if not self.send_Zemismart_packet(self.get_position_cmd, bytearray([0x01]), 1):
             return False
         elif not self.send_Zemismart_packet(self.get_battery_cmd, bytearray([0x01]), 1):
             return False
         else:
             return True
+
+    def timer_toggle(self, timer_id, enabled):

```

Comment:
"toggle" may not be the best name. When I hear the word toggle, I think of changing the enabled state to the opposite of the current enabled state. I can't think of a better name though.

Reply:
Yeah, same here. I was thinking about making two separate methods, but it seemed excessive.